__tests__/app.test.js

```diff
@@ -97,30 +73,48 @@ describe("GET /api/articles", () => {
   });
 
   test("200: responds with array of article objects sorted by created_at date property", async () => {
-      const response = await request(app).get("/api/articles").expect(200);
+    const response = await request(app).get("/api/articles").expect(200);
     const { body } = response;
 
     expect(body.article).toBeSortedBy("created_at", { descending: true });
   });
-  
+
   test("Each reply must respond with an object which includes author, title, article_id, topic, created_at, votes, article_img_url, and comment count", async () => {
     const response = await request(app).get("/api/articles").expect(200);
-      const { body } = response;
-
-      expect(body.article.length).toBe(13);
-  
-      for (const article of body.article) {
-        expect(typeof article.author).toBe("string");
-        expect(typeof article.title).toBe("string");
-        expect(typeof article.article_id).toBe("number");
-        expect(typeof article.topic).toBe("string");
-        expect(typeof article.created_at).toBe("string");
-        expect(typeof article.votes).toBe("number");
-        expect(typeof article.article_img_url).toBe("string");
-        expect(typeof article.comment_count).toBe("number");
-      }
+    const { body } = response;
+
+    expect(body.article.length).toBe(13);
+
+    for (const article of body.article) {
+      expect(typeof article.author).toBe("string");
+      expect(typeof article.title).toBe("string");
+      expect(typeof article.article_id).toBe("number");
+      expect(typeof article.topic).toBe("string");
+      expect(typeof article.created_at).toBe("string");
+      expect(typeof article.votes).toBe("number");
+      expect(typeof article.article_img_url).toBe("string");
+      expect(typeof article.comment_count).toBe("number");
+    }
   });
 });
+
+describe("GET /api/articles/:article_id/comments", () => {
+  test("receives status 200", () => {
+    return request(app).get("/api/articles/1/comments").expect(200);
+  });
+
+  test("Must respond with an array of objects which includes body, votes, author, article_id, created_at", async () => {
+    const response = await request(app).get("/api/articles/1/comments").expect(200);
+    const { body } = response;
+
+    expect(body.comments.length).toBe(11);
+
+    for (const comment of body.comments) {
+      expect(typeof comment.body).toBe("string");
+      expect(typeof comment.votes).toBe("number");
+      expect(typeof comment.author).toBe("string");
+      expect(typeof comment.article_id).toBe("number");
+      expect(typeof comment.created_at).toBe("string");
```

Comment:
what if the path is /articles/1000/comments 
or /articles/hello/comments - which error tests do we need?